docs/plugins/repository-hdfs.asciidoc

```diff
@@ -77,6 +77,11 @@ include::repository-shared-settings.asciidoc[]
     the pattern with the hostname of the node at runtime (see
     link:repository-hdfs-security-runtime[Creating the Secure Repository]).
 
+`hdfs_replication_factor`::
```

Comment:
Not _entirely_ sold on the property name - maybe something like `create_opts.replication`? That leaves the spec open to be extended in the future with other create opts if we want to, and also self-documents a bit better? I'm open to suggestions.

Reply:
Sure, I don't mind that. I've renamed it.

My only gripe is that it kind of feels like exposing internal implementation, since I assume that most users interact with HDFS using cli instead of Java client code. However, I assume that most users will have to check what it does in ES docs anyway.

Reply:
> most users interact with HDFS using cli instead of Java client code

That is a fair point - maybe just `replication_factor` then? Sorry for the back and forth 😄

Reply:
Sure! I've renamed it to `replication_factor`.

> Sorry for the back and forth 😄

No problem at all! It's much more complicated to modify after the fact, so it's good to consider all options now.